test/framework/imageregistry.go

```diff
@@ -581,7 +581,7 @@ func SetupAvailableImageRegistry(t *testing.T, spec *imageregistryapiv1.ImageReg
 }
 
 func TeardownImageRegistry(te TestEnv) {
-	defer WaitUntilClusterOperatorsAreHealthy(te, 10*time.Second, AsyncOperationTimeout)
+	defer WaitUntilClusterOperatorsAreHealthy(te, 30*time.Second, AsyncOperationTimeout)
```

Comment:
FTR this change was made because vsphere operator e2e tests were consistently failing on CI jobs, but passing locally. We figured the teardown was timing out and the change from 10 to 30 seconds confirms this suspicious (the tests are passing now)